word_guess.rb

```diff
@@ -0,0 +1,157 @@
+# pair_programming https://github.com/Ada-C7/Word-Guess
+# Elizabeth and Tamiko
+
+require 'colorize'
+
+# new class for word game
+class WordGame
+
+  def initialize
+    @win = false
+    @word = ["PIZZA", "JUMPY", "HELLO", "WORLD", "QUACK", "TAXES"].sample
+    @all_guesses = []
+    @duck_count = 5
+    @open_game = open_game
```

Comment:
Here you save the value returned from the `open_game` method as an instance variable. Is this something you need somewhere else in the class? If not, you can just call `open_game` without saving the result.